src/app/model-poller.js

```diff
@@ -1,146 +0,0 @@
-import * as Sentry from "@sentry/browser";
-import {
-  disableControllerUUIDMasking,
-  fetchAllModelStatuses,
-  fetchControllerList,
-  loginWithBakery,
-} from "juju";
-
-import {
-  storeLoginError,
-  updateControllerConnection,
-  updateJujuAPIInstance,
-  updatePingerIntervalId,
-} from "app/actions";
-
-import { updateModelList } from "juju/actions";
-
-import {
-  getConfig,
-  getControllerConnections,
-  getUserPass,
-  getWSControllerURL,
-  isLoggedIn,
-} from "./selectors";
-
-export default async function connectAndListModels(
-  reduxStore,
```

Comment:
Sorry I'm still new to Redux's ecosystem, the reason that we are moving this into a middleware is because this function basically takes as a parameter `reduxStore`?

Reply:
Ah right, sorry the context is that it is recommended that [only serialisable data](https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state) should live in the redux store and websockets should [live in the middleware](https://redux.js.org/faq/code-structure#where-should-websockets-and-other-persistent-connections-live).

My followup branch will replace the dispatches to store the controllers and models in the store and instead store them in the middleware itself.

The further context is that this is a step towards being able to construct the whole store with test factories.